int/api/websites/webcreator.go

```diff
@@ -71,6 +77,21 @@ func prepareForWebsiteHandler(params operations.WebsiteCreatorPrepareParams, app
 			})
 }
 
+func GetMaxArchiveSize() int {
+	strVar := os.Getenv(UploadMaxSize)
+
+	if strVar == "" {
+		return defaultMaxArchiveSize
+	}
+
+	intVar, err := strconv.Atoi(strVar)
+	if err != nil {
+		panic(err)
```

Comment:
As it's called from an HTTP call, we need to return an error in the

Reply:
I a pushed a commit to return the default value